service/src/test/java/bio/terra/workspace/common/fixtures/WorkspaceFixtures.java

```diff
@@ -39,6 +43,35 @@ public static void createGcpCloudContextInDatabase(
         workspaceUuid, CloudPlatform.GCP, new GcpCloudContext(projectId).serialize(), flightId);
   }
 
+  /**
+   * Creates a workspaces with a GCP cloud context and stores it in the database. Returns the
+   * workspace id.
+   *
+   * <p>The {@link ResourceDao#createControlledResource(ControlledResource)} checks that a relevant
+   * cloud context exists before storing the resource.
+   */
+  public static UUID createGcpWorkspace(WorkspaceDao workspaceDao) {
```

Comment:
Forgot to comment earlier

createGcpWorkspace -> createGcpWorkspaceWithContext

createWorkspace -> createWorkspaceWithoutContext

Reply:
Actually, there's WorkspaceConnectedTestUtils.createWorkspaceWithGcpContext(). WorkspaceConnectedTestUtils is a better place for these methods, than WorkspaceFixtures.

Use existing WorkspaceConnectedTestUtils.createWorkspaceWithGcpContext(), and add WorkspaceConnectedTestUtils.createWorkspaceWithoutGcpContext().

Reply:
i'm using this in the unit test. No real context is created by this method. 

i can rename though.

Reply:
Can you create WorkspaceUnitTestUtils and put these there? These Fixtures classes are normally for defining constants, not making calls.